src/_utils/ArrayUtils.js

```diff
@@ -1 +1,13 @@
 export const arrayHas = (arr, v) => arr.indexOf(v) > -1;
+
+export const groupByKey = (arr, key) => {
+    const result = {};
```

Comment:
This is a reducer waiting to happen, in fact I think I've written it somewhere already ;)